src/v2/stack_aggregator.py

```diff
@@ -204,6 +207,7 @@ def _get_package_details(self, component):
 
     def _get_package_details_with_vulnerabilities(self) -> List[Dict[str, object]]:
         """Get package data from graph along with vulnerability."""
+        eri = self._request.external_request_id if self._request is not None else 'UNKNOWN'
```

Comment:
Request id is a mandatory field, no need to check for null. It is been validated by the request model.

Reply:
This check is required for testing code, without this check, unit test case will fail.

Reply:
How come a unit test fails after adding debug print?

Reply:
Request object was not set properly during unit testing. Now fixed the test case and removed if-else check in main code.